lib/utils.py

```diff
@@ -54,27 +55,27 @@ def make_request(
     server is not in sync with our NTP server or local computer).
     """
     # Ensure the URL is not malformed
-    site = site.replace("//", "/").lstrip("/")
+    site = re.sub(r"(?<!:)//+", "/", site).lstrip("/")
     url = BASE_URL + site
 
     attempts = 0
     while attempts < max_attempts:
         attempts += 1
-        if method == "POST":
-            response = requests.post(url, headers=headers, json=info)
-        else:
-            response = requests.get(url, headers=headers, params=info)
+        try:
```

Comment:
Could we move this try back down to where it was? I don't believe any more errors would be caught in the except block by placing it up here.